README.md

```diff
@@ -1,58 +1,102 @@
 # invenio-subjects-lcsh
 
-LCSH subject terms for InvenioRDM
+*LCSH subject terms for InvenioRDM*
 
 Install this extension to get [Library of Congress Subject Headings](https://id.loc.gov/authorities/subjects.html) into your instance.
 
 ## Installation
 
 From your instance directory:
 
-    pipenv install invenio-subjects-lcsh
+```bash
+pipenv install invenio-subjects-lcsh
+```
 
 This will add it to your Pipfile.
 
-### Versions
+## Versions
 
-This repository follows [calendar versioning](https://calver.org/):
+This repository follows [calendar versioning](https://calver.org/) for year and month:
 
-`2021.06.18` is both a valid semantic version and an indicator of the year-month corresponding to the loaded LCSH terms.
+`2021.06.18` is both a valid semantic version and an indicator of the year-month corresponding to the loaded terms.
+`18` here is a patch number (not a day).
 
+So far the package is compatible with InvenioRDM 9.1+'s subjects "ABI". If there is a breaking change, a compatibility
+table will be provided to indicate which version is compatible with with InvenioRDM's "ABI".
 
 ## Usage
 
 There are 2 types of users for this package. Maintainers of the package and instance administrators.
 
 ### Instance administrators
 
-For instance administrators, after you have installed the extension as per the steps above, you are done. It should just work.
+For instance administrators, after you have installed the extension as per the steps above, you will want to reload your instance's fixtures: `pipenv run invenio rdm-records fixtures`. This will install the new terms in your instance.
 
-### TODO -- Maintainers
+Updating existing terms currently requires manual replacement.
 
-When a new list of LCSH term comes out, this package should be updated. Here we show how.
+### Maintainers
 
-0- Make sure you have cloned this package. Then install it locally with:
+When a new list of MeSH term comes out, this package should be updated. Here we show how.
 
-    make install
+0. Install this package locally with the `dev` extra:
 
-1- Update:
+```bash
+pipenv run pip install -e .[dev]
+```
 
-    make update
+1. Use the installed `galter-subjects-utils` tool to get the new list:
+
+```bash
+pipenv run galter-subjects-utils lcsh --output-file invenio_subjects_lcsh/vocabularies/subjects_lcsh.jsonl
+```
```

Comment:
`subjecrts_lcsh.jsonl` will be a relatively large file, no? Just something to keep an eye on if it is being committed in the project.

Reply:
Oh, I mean it is a large file (~60MB), it is already included in the project and it was updated in this PR :smile: . There isn't a zip loader in InvenioRDM core modules to load zipped files, so we can't distribute it zipped right now. We could save a lot of space by doing so if it was supported.